packages/sitecore-jss-react/tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "outDir": "dist/cjs",
     "strictNullChecks": false,
     "typeRoots": ["node_modules/@types"],
-    "declarationDir": "./types"
+    "declarationDir": "./types",
+    "skipLibCheck": true
```

Comment:
Any reason why are we enabling this? Just to shorten the compile time?

Reply:
React library throws an error, may be when we will do upgrade of the typescript - it will disappear